packages/clerk-js/src/core/resources/User.ts

```diff
@@ -80,6 +80,7 @@ export class User extends BaseResource implements UserResource {
   publicMetadata: UserPublicMetadata = {};
   unsafeMetadata: UserUnsafeMetadata = {};
   lastSignInAt: Date | null = null;
+  createOrganizationEnabled = false;
```

Comment:
Should we actually default this to false?

Reply:
It shouldn't matter in this case as a User is always initialized from json data.

You definitely have more context than me,but this seems fine!